### PR TITLE
Issue #SB-15992 fix: Added env variable for phrase app token

### PR DESCRIPTION
**Pull request by Nik720**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 1]
```diff
@@ -102,3 +102,4 @@ sunbird_bot_service_URL=/chatapi/bot
 sunbird_p1_reCaptcha_enabled={{sunbird_p1_reCaptcha_enabled|default('true')}}
 sunbird_p2_reCaptcha_enabled={{sunbird_p2_reCaptcha_enabled|default('true')}}
 sunbird_p3_reCaptcha_enabled={{sunbird_p3_reCaptcha_enabled|default('true')}}
+sunbird_phraseApp_token={{sunbird_phraseApp_token|''}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:105]: This is incorrect. Please update with default keywork

Comment (Nik720) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env]: Fixed

Comment (rjshrjndrn) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env]: what will happen if the variable is empty ? @Nik720

Comment (Nik720) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env]: If this token is empty then it will call the old method.

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env]: Resolved

**Pull request merged**